platform/android/export/export.cpp

```diff
@@ -2604,10 +2631,20 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
 			if (clean_build_required) {
 				cmdline.push_back("clean");
 			}
-			cmdline.push_back("build");
+
+			String build_type = p_debug ? "Debug" : "Release";
+			if (export_format == 1) {
+				String bundle_build_command = vformat("bundle%s", build_type);
+				cmdline.push_back(bundle_build_command);
+			} else if (export_format == 0) {
+				String apk_build_command = vformat("assemble%s", build_type);
+				cmdline.push_back(apk_build_command);
+			}
```

Comment:
You can add a `else` case here and throw an error since we'd be dealing with an unsupported format.

Reply:
I added a guard at the very beginning to protect against unsupported formats